src/native/crt.c

```diff
@@ -229,8 +229,10 @@ static void s_jni_atexit_strict(void) {
 
     if (g_memory_tracing) {
         struct aws_allocator *tracer_allocator = aws_jni_get_allocator();
-        aws_mem_tracer_destroy(tracer_allocator);
+        s_allocator = aws_mem_tracer_destroy(tracer_allocator);
```

Comment:
trivial:
~~s_allocator =~~ aws_mem_tracer_destroy(tracer_allocator);
it's weird that you're setting s_allocator here. there's no reason to do this. It gets set NULL at the end of the function.

Reply:
Still trying to debug what's happening with the sba allocator and it leads to a crash sometimes.